### PR TITLE
Personalization on Caregiver Dashboard + Dependents on Registration

### PR DESCRIPTION
**Pull request by kristinaasalgado**

### CODE REVIEW

File: bxhealth/src/components/BookAppointment/BookAppointment.js [hunk 1]
```diff
@@ -2,20 +2,35 @@
...
+    }
+  }
   render() {
     return (
       <div>
         <Card className="book-appointment">
           <CardBody>
             <img src={window._env_.PUBLIC_URL + "/images/any-virtual-health-logo.svg"} alt="AnyVirtualHealth" />
-            <CardTitle tag="h3" dangerouslySetInnerHTML={{__html: data.title}} />
+            <CardTitle tag="h3"> {this.getTitle()} </CardTitle>
```
Comment (michaelsanchez-pingidentity) [inline bxhealth/src/components/BookAppointment/BookAppointment.js:33]: A couple thoughts with this. 
1. We don't want to optimize without measurements, but typically, inline functions are quicker. It's related to how React determines if it should update the DOM.
2. Is this the only use of the getTitle() method? If so, how much clean code did we gain from it if the logic could be moved as is inside the CardTitle tag? Or even a ternary operator.
3. Not using the data files for the text means future title edits mean code edits. Content should be, where feasible, abstracted from code.

File: bxhealth/src/pages/caregiver-dashboard.js [hunk 2]
```diff
@@ -82,17 +109,38 @@ class Dashboard extends React.Component {
...
+                          <h4>Primary Provider</h4>
+                          <p>Dr. Remy Runyon<br />BXHealth, SE Illinois<br />1546 S. First St., Suite 3<br />Roebuck, IL  63456</p>
+                        </div>                   
+                      </div>
+                      }
+                      {this.state.patientFullName === "" &&
                       <div className="auth-columns">
-                        <div className="auth-column" dangerouslySetInnerHTML={{__html: data.caregiver.col_left}} />
-                        <div className="auth-column" dangerouslySetInnerHTML={{__html: data.caregiver.col_right}} />                    
+                        <div className="auth-column">
```
Comment (michaelsanchez-pingidentity) [inline bxhealth/src/pages/caregiver-dashboard.js:138]: @kristinaasalgado  same comment here about hard coding content in the code.

File: bxhealth/src/pages/caregiver-dashboard.js [hunk 3]
```diff
@@ -42,11 +44,36 @@ class Dashboard extends React.Component {
...
+        const firstName = response.givenName[0];
+        this.session.setAuthenticatedUserItem("patientFirstName", firstName, "session");
+        this.setState({
+          patientFirstName: firstName
+        });
       } else {
+        console.info("Unable to retreive patient's first name. Is your patient linked?");
         this.session.setAuthenticatedUserItem("patientFirstName", "undefined", "session");
+      } 
+      if (response.cn) {
```
Comment (michaelsanchez-pingidentity) [inline bxhealth/src/pages/caregiver-dashboard.js:56]: @kristinaasalgado  this many if() statements in a row is _**similar**_ to the "if you have 2 nested ifs in an if, review the logic". Nothing to do here, but makes me feel like we, someday, may need a method that can take a response object along with an array of keys to update, and it handles iterating the object calling setAuthenticatedUserItem(). Not worth it now. Not enough evidence to support it.

**Review by michaelsanchez-pingidentity — COMMENTED** [review]
Let me know if you have questions about, or want to discuss, the comments/feedback.

File: bxhealth/src/components/BookAppointment/BookAppointment.js (hunk 1 above)
Comment (kristinaasalgado) [reply bxhealth/src/components/BookAppointment/BookAppointment.js:33]: Chatted and reviewed. Leaving as is for now!

File: bxhealth/src/pages/caregiver-dashboard.js (hunk 3 above)
Comment (kristinaasalgado) [reply bxhealth/src/pages/caregiver-dashboard.js:54]: Added a TODO for future use cases.

File: bxhealth/src/pages/caregiver-dashboard.js (hunk 2 above)
Comment (kristinaasalgado) [reply bxhealth/src/pages/caregiver-dashboard.js:136]: Will fix!

**Review by michaelsanchez-pingidentity — APPROVED** [review]

**Pull request merged**